lib/cocoapods/installer.rb

```diff
@@ -275,8 +275,6 @@ def validate_build_configurations
     #         regenerated from scratch and any file which might not be
     #         overwritten.
     #
-    # @todo   [#247] Clean the headers of only the pods to install.
```

Comment:
I think this TODO is still relevant?

Reply:
yeap still relevant. we only had a change in which we do not delete the target support files directory for pod targets because then we could keep the `xcconfig` contents and help xcode not to rebuild stuff after each `pod install` invocation.

Reply:
Ah okay, I tried to follow the issues to see if it was still relevant but they were all closed. Will revert